internal/worker/modelworkermanager/modelworkermanager_test.go

```diff
@@ -55,6 +56,12 @@ func (s *suite) setupMocks(c *gc.C) *gomock.Controller {
 }
 
 func (s *suite) SetUpTest(c *gc.C) {
+	ctrl := s.setupMocks(c)
```

Comment:
You can't call setup mocks in setup test, as the `gc.C` isn't the same as the one that reports the errors. So what ends up happening, is we miss the mock tests.

The solution is to call setup mocks in every test.

Reply:
I see, refactored this with thanks Simon